src/test/scala/ironoxide/FullIntegrationTest.scala

```diff
@@ -163,18 +168,32 @@ class FullIntegrationTest extends AsyncWordSpec with Matchers with EitherValues
     "roundtrip through a user" in {
       val sdk = IronSdkSync[IO](createDeviceContext)
       val data = ByteVector(List(1, 2, 3).map(_.toByte))
-      val result = sdk.advanced.documentEncryptUnmanaged(data, DocumentEncryptOpts()).attempt.unsafeRunSync.value
+      val encryptResult = sdk.advanced.documentEncryptUnmanaged(data, DocumentEncryptOpts()).attempt.unsafeRunSync.value
 
-      result.id.id.length shouldBe 32
-      result.encryptedDeks.bytes.isEmpty shouldBe false
+      encryptResult.id.id.length shouldBe 32
+      encryptResult.encryptedDeks.bytes.isEmpty shouldBe false
 
       val decrypt =
-        sdk.advanced.documentDecryptUnmanaged(result.encryptedData, result.encryptedDeks).attempt.unsafeRunSync.value
+        sdk.advanced
+          .documentDecryptUnmanaged(encryptResult.encryptedData, encryptResult.encryptedDeks)
+          .attempt
+          .unsafeRunSync
+          .value
 
-      decrypt.id.id shouldBe result.id.id
+      decrypt.id.id shouldBe encryptResult.id.id
       decrypt.decryptedData shouldBe data
       decrypt.accessVia.id shouldBe primaryTestUserId.id
       decrypt.accessVia shouldBe a[UserId]
+
+      //Now let's clear byte arrays and verify that they clear in the ByteVector
+      clearBytes(decrypt.underlyingBytes)
```

Comment:
Should we break these out into their own tests?

Reply:
Because it requires the value to be coming from the Java type, that's a pain. That's why I didn't do it.